packages/metal-dom/src/domNamed.js

```diff
@@ -296,6 +297,28 @@ export function exitDocument(node) {
 }
 
 /**
+ * Search by a custom event already registered and returns its configuration object.
+ * @param {!string} eventName The custom event name used to find the configuration
+ * object as an object key or by executing a regular expression.
+ * @return {Object} The custom event configuration object or undefined if the given
+ * eventName doesn't match with any custom event name.
+ */
+function getCustomEventData(eventName) {
+	var customConfig = customEvents[eventName];
+	if (!customConfig) {
+		for (var i = customUnamedEvents.length; i--; ) {
```

Comment:
I think it could be since they're regexes, so different custom events could actually match the same event name. From profiling Metal.js I can see that looping arrays haven't made a difference to performance, and this array is going to be usually very small (almost always with only one element for now even, unless a developer adds his own custom event), so going backwards shouldn't make a difference.